packages/env/lib/env.js

```diff
@@ -477,20 +474,30 @@ async function configureWordPress( environment, config ) {
 	}
 
 	// Activate all plugins.
-	for ( const pluginSource of config.pluginSources ) {
-		await dockerCompose.run(
-			environment === 'development' ? 'cli' : 'tests-cli',
-			`wp plugin activate ${ pluginSource.basename }`,
-			options
-		);
+	await dockerCompose.run(
+		environment === 'development' ? 'cli' : 'tests-cli',
+		'wp plugin activate --all',
+		options
+	);
+
+	// Get the list of themes available to WordPress.
+	const themeOutput = await dockerCompose.run(
+		environment === 'development' ? 'cli' : 'tests-cli',
+		'wp theme list --fields=name',
+		options
+	);
+
+	// In case of an error getting the themes, do nothing.
+	if ( themeOutput.exitCode !== 0 ) {
+		return;
 	}
 
-	// Activate the first theme.
-	const [ themeSource ] = config.themeSources;
-	if ( themeSource ) {
+	// Index 0 contains "name" (the header of the output)
+	const firstTheme = themeOutput.out.split( '\n' )[ 1 ];
```

Comment:
Could use destructuring here if you wanted to remove that magic number—I hear it's all the rage 😎

```suggestion
	const [ , firstTheme ] = themeOutput.out.split( '\n' );
```

Reply:
😎 I'll drink that javascript juice 🍹